src/legacy/status_im/ui/screens/signing/views.cljs

```diff
@@ -337,8 +329,8 @@
                   mainnet? [:mainnet?]
                   prices-loading? [:prices-loading?]
                   management-enabled? [:wallet-legacy/transactions-management-enabled?]]
-    (let [display-symbol     (wallet.utils/display-symbol token)
-          fee-display-symbol (wallet.utils/display-symbol (tokens/native-currency chain))]
+    (let [display-symbol     nil
+          fee-display-symbol nil]
```

Comment:
acceptable ??

Reply:
Why make it nil? If the screen is still accessible shouldn't we keep it as it is, otherwise remove it completely?

Reply:
I will be removing signing in a pr after this. The old wallet is completely non functional now so the user should not get to a signing screen with the old wallet.